tests/user.test.js

```diff
@@ -393,7 +393,7 @@ describe('THE USER TEST SUITE', () => {
 
     it('Should successfully update username when valid token is supplied', (done) => {
       const requestObject = { username: 'bodunde' };
-      request.put(`${singleRequestRoute}/2`)
+      request.put(`${singleRequestRoute}/${adminID}`)
```

Comment:
@knite51 Here's the fix.
Got a `400` status code cos it was a bad request/syntax, so was able to narrow it down here.